buildchow/templates/buildflow-template.yaml

```diff
@@ -0,0 +1,63 @@
+- job-template:
+    name: 'jbosstools-buildflow_{stream}'
+    disabled: '{obj:disabled}'
+    description: 'Build Flow that replaces the need for job cascading and interlinkage. Runs weekly. See <a href="https://issues.jboss.org/browse/JBIDE-13307">JBIDE-13307</a>'
+    jdk: '{default-jdk}'
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 10
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+    project-type: flow
+    dsl: |
+      parallel (
+      {{ ignore(FAILURE) {{ build("jbosstools-aerogear_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-base_{stream}") }} }}
+      )
+      parallel (
+      {{ 
+      parallel (
+      // {{ ignore(FAILURE) {{ build("jbosstools-gwt_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-forge_{stream}") }} }},
+      // {{ ignore(FAILURE) {{ build("jbosstools-freemarker_{stream}") }} }}, // unchanged for master branch so far
+      {{ 
+      ignore(FAILURE)  {{ build("jbosstools-server_{stream}") }}
+      parallel (
+      {{ ignore(FAILURE) {{ build("jbosstools-livereload_{stream}") }} }},
```

Comment:
Why did you double up all the curly braces? This looks nothing like what I have in https://jenkins.mw.lab.eng.bos.redhat.com/hudson/view/DevStudio/view/DevStudio_9.0.mars/job/jbosstools-buildflow_master/configure